### ISSUE TITLE
Data  and Hyper-parameter Details Inquiry

### ISSUE THREAD

**Issue by xvrong** [WangYZ1608/Self-Prompt-Tuning#4]
Your work looks great, and I am interested in replicating it. I have two questions that I need your help.

The first question concerns the data. I found the vtab-1k dataset in [this repository](https://github.com/dongzelian/SSF). Are your data and splitting files the same as those?

The second question relates to hyper-parameters. In issue #1, you mentioned that:
> Different tasks require setting different hyper-parameters, including learning rate (LR) and weight decay (WD).

Do you have any plans to share the hyper-parameters or scripts for the tasks mentioned in the paper?

Thank you.
